packages/app-project/stores/User/UserPersonalization/Notifications/helpers/getTalkUnreadConversationsIds.js

```diff
@@ -0,0 +1,31 @@
+import { talkAPI } from '@zooniverse/panoptes-js'
+
+export default async function getUnreadConversationsIds (authorization) {
+  if (!authorization) return undefined
```

Comment:
```suggestion
  if (!authorization) return []
```
This function returns an array of conversations, so I’d suggest returning an empty array here.